src/openforms/submissions/tests/factories.py

```diff
@@ -56,7 +69,7 @@ class Params:
                 lambda s: s.completed_on - timedelta(hours=4)
             ),
             pre_registration_completed=True,
-            price=factory.PostGenerationMethodCall("calculate_price"),
```

Comment:
`PostGenerationMethodCall` is somewhat counterintuitive to use. Calling:

`SubmissionFactory.create(completed=True, price=Decimal("1"))` will not result in `submission.price == Decimal("1")`. Instead, `Decimal("1")` is passed to the `calculate_price` method as a positional argument. For the record, here is the signature:

```python
def calculate_price(self, save=False): ...  # save would take the value of `Decimal("1")` with the example above.
```

By switching to `factory.PostGenerationMethodCall("calculate_price", save=False)`, it would lead to a test failure where `SubmissionFactory.create(completed=True, price=Decimal("1"))` was used. The test in question was accidentally still passing :o.

Went for `PostGeneration` instead which is more verbose but more intuitive.